webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -86,7 +86,7 @@ msgstr "Kontakt - Der Steuerlotse für Rente und Pension"
 
 #: app/routes.py:237
 msgid "imprint.header-title"
-msgstr "Impressum - Der Steuerlotse für Rente und Pension"
+msgstr "Kontakt - Der Steuerlotse für Rente und Pension"
```

Comment:
Was this on purpose? If so, why do we need to change this?

Reply:
That was not on purpose. Something must have gotten mixed up in the merge conflict.